### DISCUSSION TITLE
How to create manifest.txt for qiime2 in a nextflow?

### DISCUSSION THREAD

**Discussion by liubingdong** [nextflow-io/nextflow#3750] [Q&A]
Hello, I want to use the nextflow to construct a pipeline for my qiime2 workflow.
However, it is hard to create a manifest.txt for the qiime tools import process.
The manifest.txt looks like this
<img width="1213" alt="image" src="https://user-images.githubusercontent.com/26839870/224874862-dc2801f6-6a19-4689-8b53-b9bece17ddba.png">
Anyone could help me to construct this file in nextflow process? Thanks!

**Comment by irvinng98** [comment]
Hi I have the same issue and was wondering if you ever figured it out. Thanks.
